hardware/Meteorologisk.cpp

```diff
@@ -0,0 +1,446 @@
+#include "stdafx.h"
+#include "Meteorologisk.h"
+#include "../main/Helper.h"
+#include "../main/Logger.h"
+#include "../httpclient/UrlEncode.h"
+#include "hardwaretypes.h"
+#include "../main/localtime_r.h"
+#include "../httpclient/HTTPClient.h"
+#include "../main/json_helper.h"
+#include "../main/RFXtrx.h"
+#include "../main/mainworker.h"
+#include "../main/SQLHelper.h"
+
+#define round(a) ( int ) ( a + .5 )
+
+#ifdef _DEBUG
+#define DEBUG_MeteorologiskR
```

Comment:
Please comment the two debug lines as some are building in Debug mode

Reply:
Done